cmd/esbuild/main.go

```diff
@@ -635,6 +637,17 @@ func run(fs fs.FS, args argsObject) {
 		}
 		args.logInfo(fmt.Sprintf("Wrote to %s (%s)", path, toSize(len(item.JsContents))))
 
+		// Write out the additional files
+		for _, file := range item.AdditionalFiles {
```

Comment:
The contents of `AdditionalFiles` end up being written out twice because they are both written out here and duplicated to become normal files in this code elsewhere:

```go
		for _, bundle := range group {
			for _, additionalFile := range bundle.AdditionalFiles {
				if additionalFile.Path != "" {
					results = append(results, BundleResult{additionalFile.Path, []byte(additionalFile.Contents), "", []byte(""), []AdditionalFile{}})
				}
			}
		}
```

That's ok, I can fix it after landing this.